CONTRIBUTING.md

```diff
@@ -238,6 +238,120 @@ installer-test-rhel-v7-install-default> Formatting './disk.qcow2', fmt=qcow2 clu
 
 </details>
 
+## Container tests
+
+
+In `nix/tests/container-test` there exists some Nix derivations which we expose in the flake via `hydraJobs`.
+
+These should be visible in `nix flake show`:
+
+```
+❯ nix flake show
+warning: Git tree '/home/ana/git/determinatesystems/nix-installer' is dirty
+git+file:///home/ana/git/determinatesystems/nix-installer
+# ...
+├───hydraJobs
+│   ├───container-test
+│   │   ├───all
+│   │   │   └───x86_64-linux
+│   │   │       ├───all: derivation 'all'
+│   │   │       ├───docker: derivation 'all'
+│   │   │       └───podman: derivation 'all'
+│   │   ├───ubuntu-v18_04
+│   │   │   └───x86_64-linux
+│   │   │       ├───all: derivation 'all'
+│   │   │       ├───docker: derivation 'vm-test-run-container-test-ubuntu-v18_04'
+│   │   │       └───podman: derivation 'vm-test-run-container-test-ubuntu-v18_04'
+│   │   ├───ubuntu-v20_04
+│   │   │   └───x86_64-linux
+│   │   │       ├───all: derivation 'all'
+│   │   │       ├───docker: derivation 'vm-test-run-container-test-ubuntu-v20_04'
+│   │   │       └───podman: derivation 'vm-test-run-container-test-ubuntu-v20_04'
+│   │   └───ubuntu-v22_04
+│   │       └───x86_64-linux
+│   │           ├───all: derivation 'all'
+│   │           ├───docker: derivation 'vm-test-run-container-test-ubuntu-v22_04'
+│   │           └───podman: derivation 'vm-test-run-container-test-ubuntu-v22_04'
+```
+
+To run all of the currently supported tests:
+
+```bash
+nix build .#hydraJobs.container-test.all.x86_64-linux.all -L
+```
+
+To run a specific distribution listed in the `nix flake show` output:
+
+```bash
+nix build .#hydraJobs.container-test.ubuntu-v22_04.x86_64-linux.docker -L
+```
+
+For PR review, you can also test arbitrary branches or checkouts like so:
+
+```bash
+nix build github:determinatesystems/nix-installer/${BRANCH}#hydraJobs.container-test.ubuntu-v22_04.x86_64-linux.podman -L
```

Comment:
Maybe not a problem since most of the work is currently happening in this repo, but if we see a lot of external contributors, this won't work for them, because branches aren't global. (However, git shas are shared, and they could use that.)

Not really something that needs to be addressed, but to keep in mind.

Reply:
Ya you're right, hmmm...